scripts/murmur.ini

```diff
@@ -15,6 +15,25 @@
 ; murmur.sqlite in default locations or create it if not found.
 database=
 
+; Murmur defaults to using SQLite with its default rollback journal.
+; In some situations, using SQLite's write-ahead log can be advantageous.
```

Comment:
This may fuck up the formatting, but explicitly mentioning the WAL acronym when using "write-ahead log" could make the acronym usage below more intuitively understandable.